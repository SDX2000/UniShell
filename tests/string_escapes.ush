#!/usr/bin/env unishell
echo "================"
echo $SCRIPT_NAME
echo "================"

echo "a\tb\n"