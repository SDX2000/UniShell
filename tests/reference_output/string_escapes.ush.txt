================
string_escapes.ush
================
a	b

